### ISSUE TITLE
Quick fix all for abstract class creates duplicates in the dropdown and code

### ISSUE THREAD

**Issue by AlCalzone** [Microsoft/vscode#43181]
### Issue Type
Bug

### Description

When extending an abstract class with multiple methods/properties, the quick fix dropdown has multiple instances of the same solution. In addition, selecting "quick fix all" creates duplicates of the function implementations.

## Steps to reproduce:
1. Create a TypeScript file with contents
```ts
export abstract class Foo {
	abstract a(): void;
	abstract b(): void;
	abstract c(): void;
}

class Bar extends Foo {
	
}
```

2. place the cursor on `Bar`, open the quick fix dropdown
![foo1](https://user-images.githubusercontent.com/17641229/35965124-2d562c04-0cba-11e8-989b-10600b48ae5b.png)

3. select "quick fix all"
![foo2](https://user-images.githubusercontent.com/17641229/35965126-2f975704-0cba-11e8-9507-902bf077a5bd.PNG)

### VS Code Info

VS Code version: Code 1.20.0 (c63189deaa8e620f650cc28792b8f5f3363f2c5b, 2018-02-07T17:10:31.920Z)
OS version: Windows_NT ia32 6.1.7601

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6600 CPU @ 3.30GHz (4 x 3312)|
|Memory (System)|7.93GB (3.49GB free)|
|Process Argv|C:\Program Files (x86)\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
java|red|0.18.1
vscode-java-debug|vsc|0.6.0
vscode-java-pack|vsc|0.2.0

</details>
Reproduces without extensions

**Comment by mjbvz** [comment]
Thanks. Fixed the duplicate quick fixes showing in the light bulb menu with d2d18a4

The duplicate method generation looks like an upstream issue. I've opened  https://github.com/Microsoft/TypeScript/issues/21776 to track this

**Comment by mjbvz** [comment]
Note to February verifier: please only check that duplicate entries do not show up in the lightbulb menu. The bug with applying the quick fix itself is an upstream issue and I'm not sure when TS will get around to fixing it
